packages/shared/src/graphql/squads.ts

```diff
@@ -56,9 +69,79 @@ export const CREATE_SQUAD_MUTATION = gql`
 `;
 
 export const ADD_POST_TO_SQUAD = gql`
-mutation AddPostToSquad($data: AddPostToSquadInput) {
-  sharePost(data: $data) {
+  mutation AddPostToSquad($data: AddPostToSquadInput) {
+    sharePost(data: $data) {
       Post!
     }
   }
 `;
+
+export const SQUAD_QUERY = gql`
+  query Source($handle: ID!) {
+    source(id: $handle) {
+      id
+      active
+      handle
+      name
+      permalink
+      public
+      type
+      description
+      image
+    }
+  }
+`;
+
+export const SQUAD_MEMBERS_QUERY = gql`
+  query Source($id: ID!) {
+    sourceMembers(first: 5, sourceId: $id) {
+      edges {
+        node {
+          role
+          user {
+            id
+            name
+            image
+            permalink
+            username
+            bio
+          }
+        }
+      }
+    }
+  }
+`;
+
+export type SquadData = {
+  source: Squad;
+};
+
+export type SquadEdgesData = {
+  sourceMembers: {
+    edges: Edge<SquadMember>[];
+  };
+};
+
+export async function getSquad(handle: string): Promise<Squad> {
+  const res = await request<SquadData>(`${apiUrl}/graphql`, SQUAD_QUERY, {
+    handle,
+  });
+  return res.source;
+}
```

Comment:
I like this! Makes the function calls centralized and reduced the verbosity of accessing the property applied by the GraphQL request 🚀 

I think we should continue doing this in the future.